pkg/errors/error.go

```diff
@@ -33,6 +33,7 @@ var (
 	ErrPeerLocalNode           = errors.New("The peer being added is the local node")
 	ErrProcessNotFound         = errors.New("The process is not running or is inaccessible")
 	ErrProcessAlreadyRunning   = errors.New("Process is already running")
+	ErrProcessNotRunning       = errors.New("Process is not running")
```

Comment:
Recently, @Madhu-1 sent a PR to make all error strings to start with small caps. You can do that here.

Reply:
done